src/containers/BooksList.js

```diff
@@ -1,29 +1,43 @@
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
+import { bookRemoveAction } from '../actions/index';
 
 const BookList = (props) => {
   const { books } = props;
 
```

Comment:
- [ ] Consider destructuring your props in a single line, this makes the code look more clean and professional.
       ` const { books , removeBook} = props   OR  const BookList = ({books, removeBook}) => {....}`